htsohm/db/material.py

```diff
@@ -143,155 +43,22 @@ def __init__(self, run_id=None, ):
-            if 'gas_adsorption' in simulations:
-                ga_limits = simulations['gas_adsorption']['limits']
-                if isinstance(simulations['gas_adsorption']['external_pressure'], list):
-                    ga1_o = self.ga1_absolute_volumetric_loading    # initally-calculated values
-                    ga1_mean = self.retest_gas_adsorption_1_sum / self.retest_num
-                else:
-                    ga1_o = 0
-                    ga1_mean = 0
-            elif 'artificial_gas_adsorption' in simulations:
-                ga_limits = simulations['artificial_gas_adsorption']['limits']
-                ga1_o = 0
-                ga1_mean = 0
-            ga_width = (ga_limits[1] - ga_limits[0]) / number_of_bins
-        else:
-            ga0_o = 0
-            ga0_mean = 0
-            ga_width = 0
-
-        if 'surface_area' in simulations or 'artificial_surface_area' in simulations:
-            sa_o = self.sa_volumetric_surface_area
-            sa_mean = self.retest_surface_area_sum / self.retest_num
-            if 'surface_area' in simulations:
-                sa_limits = simulations['surface_area']['limits']
-            if 'artificial_surface_area' in simulations:
-                sa_limits = simulations['artificial_surface_area']['limits']
-            sa_width = (sa_limits[1] - sa_limits[0]) / number_of_bins
-        else:
-            sa_o = 0
-            sa_mean = 0
-            sa_width = 0
-
-        if 'helium_void_fraction' in simulations or 'artificial_void_fraction' in simulations:
-            vf_o = self.vf_helium_void_fraction
-            vf_mean = self.retest_void_fraction_sum / self.retest_num
-            if 'helium_void_fraction' in simulations:
-                vf_limits = simulations['helium_void_fraction']['limits']
-            if 'artificial_void_fraction' in simulations:
-                vf_limits = simulations['artificial_void_fraction']['limits']
-            vf_width = (vf_limits[1] - vf_limits[0]) / number_of_bins
-        else:
-            vf_o = 0
-            vf_mean = 0
-            vf_width = 0
-
-        retest_failed = (
-            abs(ga0_mean - ga0_o) >= tolerance * ga_width and ga0_o != 0 or
-            abs(ga1_mean - ga1_o) >= tolerance * ga_width and ga1_o != 0 or
-            abs(sa_mean - sa_o) >= tolerance * sa_width and sa_o != 0 or
-            abs(vf_mean - vf_o) >= tolerance * vf_width and vf_o != 0
-        )
-
-        return not retest_failed
+#    @property
+#    def bin(self):
+#        """Determine material's structure-property bin.
+#
+#        Args:
+#            self (class): row in material table.
+#
+#        Returns:
+#            The bin corresponding to a material's gas loading, void
+#            fraction, and surface area data and their postion in this three-
+#            dimension parameter-space.
+#
+#        """
+#        binning = [*[e[0] for e in session.query(GasLoading.bin).filter(GasLoading.material_id == self.id).all()]
```

Comment:
to be changed after conversation about how bins will work

Reply:
commit 823f9a845e6a8505742e12ac4b345d3c09867c83
Author: akaija <a.r.kaija@gmail.com>
Date:   Wed Jun 20 16:01:14 2018 -0400

    removed commented-out code